mission-control-client/src/utils/axiosLabsGraphQL.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const axiosLabsGraphQL = axios.create({
+  baseURL: process.env.REACT_APP_LABS_API_URL,
```

Comment:
I don't think it's necessary to  have the URL in the environment, but it's not a horrible thing to have.